web/src/auth/localAuth.ts

```diff
@@ -42,59 +42,64 @@ export const localAuthClient = {
     return localStorage.getItem('local_auth_token')
   },
   getUserMetadata: async () => {
-    if (!localStorage.getItem('local_auth_token')) {
-      return null
+    return {
```

Comment:
This function _must_ return a not-null value in order for this to work. So we can either return some valid hard-coded value as seen in this PR or something that looks like the following:
```
{
    email: localStorage.getItem('local_auth_token'),
}
```